src/theme.js

```diff
@@ -437,6 +435,7 @@ export const GlobalStyle = createGlobalStyle`
     line-height: 1.4;
     margin: 2rem 0;
     font-weight: 700;
+    scroll-margin-top: ${theme.variables.navHeight};
```

Comment:
```suggestion
    scroll-snap-margin: ${theme.variables.navHeight};
    scroll-margin-top: ${theme.variables.navHeight};
```

Reply:
I tried using this and it wasn't working. Maybe we can make an issue to change over, but would like to see this get in for now if thats ok with you.